assets/js/sections/simplefle.js

```diff
@@ -363,14 +446,32 @@ function handleInput() {
 				sotaWwffText = `W: ${sotaWwff}`;
 			}
 
+			// Contest exchange info: sent
+			let stx_info = "";
+			if (stx != '') {
+				stx_info += `<span data-bs-toggle="tooltip" class="badge text-bg-light">${stx}</span>`;
+			}
+			if (add_info.stx_string !== undefined && add_info.stx_string.length > 0) {
+				stx_info += `<span data-bs-toggle="tooltip" class="badge text-bg-light">${add_info.stx_string}</span>`;
+			}
+
+			// Contest exchange info: received
+			let srx_info = "";
+			if (srx != '') {
+				srx_info += `<span data-bs-toggle="tooltip" title="" class="badge text-bg-light">${srx}</span>`;
+			}
+			if (add_info.srx_string !== undefined && add_info.srx_string.length > 0) {
+				srx_info += `<span data-bs-toggle="tooltip" title="" class="badge text-bg-light">${add_info.srx_string}</span>`;
+			}
+
 			const tableRow = $(`<tr>
 			<td>${extraQsoDate}</td>
 			<td>${qsotime}</td>
 			<td>${callsign}</td>
 			<td><span data-bs-toggle="tooltip" data-placement="left" title="${freq}">${band}</span></td>
 			<td>${mode}</td>
-			<td>${rst_s}</td>
-			<td>${rst_r}</td>
+			<td>${rst_s}${stx_info}</td>
+			<td>${rst_r}${srx_info}</td>
 			<td>${gridsquare}</td>
```

Comment:
Please add some spacing here
```suggestion
			<td>${rst_s} ${stx_info}</td>
			<td>${rst_r} ${srx_info}</td>
```

Reply:
And you may want to update the SimpleFLE documentation when this PR is merged ;-) but its fine after merge. Would really appreciate it :)

https://github.com/wavelog/wavelog/wiki/SimpleFLE